jax/api.py

```diff
@@ -640,13 +642,19 @@ def pmap(fun, axis_name=None):
   like all-reduce sum.
 
   The mapped axis size must be less than or equal to the number of XLA devices
-  available. For nested ``pmap`` calls, the product of the mapped axis sizes
-  must be less than or equal to the number of XLA devices.
+  available (unless `devices` is specified, see below). For nested ``pmap``
+  calls, the product of the mapped axis sizes must be less than or equal to the
+  number of XLA devices.
 
   Args:
     fun: Function to be mapped over argument axes.
     axis_name: Optional, a hashable Python object used to identify the mapped
       axis so that parallel collectives can be applied.
+    devices: Optional, a sequence of Devices to map over. (Available devices can
```

Comment:
Should we indicate this as an experimental API likely to change (even if it's not likely to change, just to provide some cover)?